Data-Science/Kubeflow/Financial-Time-Series/financial_time_series_example.ipynb

```diff
@@ -74,12 +74,11 @@
     "######################################### E2E Demo example precondition ##############################################\n",
     "\n",
     "# You need to generate secret to have ability read datasource files from S3 bucket by Spark application (Airflow DAG)\n",
-    "# Copy template \"object_store_secret.yaml.tpl\" from shared/ezua-tutorials/current-release/Data-Analytics/Spark directory to e.g. user folder\n",
+    "# Copy template \"object_store_secret.yaml.tpl\" from shared/aie-tutorials/current-release/Data-Analytics/Spark directory to e.g. user folder\n",
     "\n",
-    "import kfp\n",
+    "import os\n",
     "\n",
-    "kfp_client = kfp.Client()\n",
-    "namespace = kfp_client.get_user_namespace()\n",
+    "namespace = os.getenv('NOTEBOOK_NAMESPACE')\n",
```

Comment:
lets do such changes in a separate PR - there are more places, where `kfp_client.get_user_namespace()` workaround should be replaced.

Reply:
@umka1332 I excluded these changes from the PR